pages/testimonials.html

```diff
@@ -0,0 +1,375 @@
+---
+layout: page-fullwidth
+permalink: /testimonials/
+title: Testimonials from Workshop Attendees
+excerpt: testimonials about The Carpentries from our community
+---
+
+<h3>
+    Ethan White
+</h3>
+
+<blockquote class="testimonal">"Software Carpentry training has had a tremendous influence on the productivity of the students in my lab and several other labs on campus...
```

Comment:
small typo in the class name (here and below): 
```suggestion
<blockquote class="testimonial">"Software Carpentry training has had a tremendous influence on the productivity of the students in my lab and several other labs on campus...
```